skype_sender_prod.py

```diff
@@ -36,75 +36,76 @@ def remove_html_tags(text):
-    if request.method == 'POST':
-        app.logger.info('/send telegram request:' + "".join(request.data.splitlines()))
-        _json = request.get_json(force=True)
+    if _json["event"]["type"] == "message" and 'subtype' not in _json["event"]:
+        if _json["production"]:
+            clusterChat=clusterChat_prod
 
-        if _json["event"]["type"] == "message":
-            if not 'subtype' in _json["event"]:
+        sk = Skype(skypeLogin, skypePass, 'session.tmp')  # connect to Skype
+        chats = sk.chats  # your conversations
+        ch = chats.chat(clusterChat)
 
+        r = requests.get(slackGetUserUrl, params={"token": slackToken, "user": _json["event"]["user"]})
+        # print r.content
+        _user = jsonpickle.decode(r.content)
+        # ch.sendMsg( _user["user"]["real_name"] + ' via Slack: ' + _json["event"]["text"]) # plain-text message
+        ch.sendRaw(messagetype="RichText", contenttype="text",
+                   content='<b>' + _user["user"]["real_name"] + ' via Slack: </b>' + remove_html_tags(_json["event"][
+                       "text"]))  # plain-text message
+        app.logger.info(_user["user"]["real_name"] + ' via Slack: ' + remove_html_tags(_json["event"]["text"]))
 
-                sk = Skype(skypeLogin, skypePass, 'session.tmp')  # connect to Skype
-                chats = sk.chats  # your conversations
-                ch = chats.chat(clusterChat)
+        return '{"OK":true}'
 
-                ch.sendRaw(messagetype="RichText", contenttype="text",
-                           content='<b>' + _json["event"]["user"] + ' via Telegram: </b>' + remove_html_tags(_json["event"][
-                               "text"]))  # plain-text message
-                app.logger.info(_json["event"]["user"] + ' via Telegram: ' + remove_html_tags(_json["event"]["text"]))
-                return '{"OK":true}'
+    return '{"OK":true}'
+    
 
+#API call to send Skype message from Telegram, does login and sends message, uncomment print statesments to debug
+@app.route('/listen/send_skype_telegram', methods=['GET', 'POST'])
+def send_skype_telegram():
+    if request.method != 'POST':
+        return
+
+    app.logger.info('/send telegram request:' + "".join(request.data.splitlines()))
+    _json = request.get_json(force=True)
+
+    if _json["event"]["type"] == "message" and 'subtype' not in _json["event"]:
+    
+        sk = Skype(skypeLogin, skypePass, 'session.tmp')  # connect to Skype
+        chats = sk.chats  # your conversations
+        ch = chats.chat(clusterChat)
+
+        ch.sendRaw(messagetype="RichText", contenttype="text",
+                   content='<b>' + _json["event"]["user"] + ' via Telegram: </b>' + remove_html_tags(_json["event"][
+                       "text"]))  # plain-text message
+        app.logger.info(_json["event"]["user"] + ' via Telegram: ' + remove_html_tags(_json["event"]["text"]))        
         return '{"OK":true}'
 
+    return '{"OK":true}'    

```

Comment:
Function `send_skype_telegram` refactored with the following changes:

- Add guard clause ([last-if-guard](https://docs.sourcery.ai/refactorings/last-if-guard/))
- Merge nested if conditions ([merge-nested-ifs](https://docs.sourcery.ai/refactorings/merge-nested-ifs/))
- Simplify logical expression using De Morgan identities ([de-morgan](https://docs.sourcery.ai/refactorings/de-morgan/))